### PR TITLE
[zh-cn] Update 'Roles and Teams'

### PR DESCRIPTION
**Pull request by jasonren0403**
### Motivation

Complete meta docs

### CODE REVIEW

File: files/zh-cn/mdn/community/roles_teams/index.md [hunk 1]
```diff
@@ -5,111 +5,277 @@ slug: MDN/Community/Roles_teams
...
+| [**维护者**](#维护者)     | 至少一个：<br>- 担任特邀专家至少六个月<br>- 了解多个项目领域<br>- 为项目的整体健康采取行动<br>- 指导其他贡献者<br>- 每月至少在项目中花费 16 小时<br>- 出席社区会议 | 批准并合并拉取请求                                     |
+| [**所有者**](#所有者)     | 仅限 Mozilla 员工                                                                                                                                                  | - 管理各种仓库的不同角色权限<br>- 添加或归档仓库和项目 |
+
+## 特殊的角色
+
+有些贡献者角色的职责更为细化，并有特殊的资格条件。这些角色包括[杰出贡献者](#杰出贡献者)、[特邀专家](#特邀专家)和[社区管理员](#社区管理员)。
+
+### 杰出贡献者
+
+杰出贡献者是那些对 MDN Web 文档做出了卓越贡献的人。他们的贡献形式包括：提出拉取请求以改进项目；在各种[交流渠道](/zh-CN/docs/MDN/Community/Communication_chanels)或学习论坛上帮助社区成员；或就 GitHub 问题和拉取请求提供反馈。
```
Comment (jasonren0403) [inline files/zh-cn/mdn/community/roles_teams/index.md:141]:
```suggestion
杰出贡献者是那些对 MDN Web 文档做出了卓越贡献的人。他们的贡献形式包括：提出拉取请求以改进项目；在各种[沟通渠道](/zh-CN/docs/MDN/Community/Communication_channels)或学习论坛上帮助社区成员；或就 GitHub 问题和拉取请求提供反馈。
```

File: files/zh-cn/mdn/community/roles_teams/index.md [hunk 2]
```diff
@@ -5,111 +5,277 @@ slug: MDN/Community/Roles_teams
...
-我们使用 [GitHub 团队](https://docs.github.com/zh/organizations/organizing-members-into-teams/about-teams)功能来管理团队。这意味着，当你被加入到团队中时，你已经传达了更密切地参与项目的意图。这也意味着你有一些额外的责任和权利：
+如果你是新人，并且想成为一名贡献者，可以看看我们的[贡献者指南](https://github.com/mdn/content/blob/main/CONTRIBUTING.md)和 [MDN GitHub 组织中的仓库列表](https://github.com/orgs/mdn/repositories)。
 
-1. 团队中的成员通常会被添加到上述相关领域的 [CODEOWNERS](https://github.com/mdn/content/blob/main/.github/CODEOWNERS) 文件中（注：对于本地化团队，则[在此处](https://github.com/mdn/translated-content/blob/main/.github/CODEOWNERS)）。
-2. 基于 CODEOWNERS 文件，当拉取请求（PR）涉及到你职责范围内的文件时，GitHub [负载均衡算法](https://docs.github.com/zh/organizations/organizing-members-into-teams/managing-code-review-settings-for-your-team#routing-algorithms) 会自动将拉取请求分配给你。
-3. 团队成员具有更高的仓库访问权限。仅为成员分配需要的仓库的访问权限。
+作为一名贡献者，你可以通过参与以下活动来参与该项目：
 
-### 如何加入团队
+- 在[交流渠道](/zh-CN/docs/MDN/Community/Communication_channels)中参与社区讨论。
```
Comment (jasonren0403) [inline files/zh-cn/mdn/community/roles_teams/index.md:28]:
```suggestion
- 在[沟通渠道](/zh-CN/docs/MDN/Community/Communication_channels)中参与社区讨论。
```

File: files/zh-cn/mdn/community/roles_teams/index.md [hunk 3]
```diff
@@ -5,111 +5,277 @@ slug: MDN/Community/Roles_teams
...
+- 组织和开展社区活动。
+- 组织与社区相关的项目会议。
+- 确定推广 MDN 项目的媒体策略。
+- 确定并实施贡献者入职体验。
+- 对新的贡献者和用户进行入职培训。
+- 确保 MDN 项目和所有参与者的健康和福利。
+- 确定并协助实施自动化，以提高项目的可持续性。
+- 会见并确保与贡献者和合作伙伴保持健康的关系。
+- 在适当的情况下，协助问题分流和拉动请求审查。
+- 监控所有[交流渠道](/zh-CN/docs/MDN/Community/Communication_channels)。
```
Comment (jasonren0403) [inline files/zh-cn/mdn/community/roles_teams/index.md:188]:
```suggestion
- 监控所有[沟通渠道](/zh-CN/docs/MDN/Community/Communication_channels)。
```

File: files/zh-cn/mdn/community/roles_teams/index.md [hunk 4]
```diff
@@ -5,111 +5,277 @@ slug: MDN/Community/Roles_teams
 
 {{MDNSidebar}}
 
-MDN Web 文档的成功和发展在很大程度上归功于我们的贡献者社区。还有致力于整个项目健康、发展和维护的正式、非正式员工，以及合作伙伴。还有一些人投入了一部分时间来协助完成 MDN Web 文档的日常工作，他们是我们[特邀专家计划](#特邀专家计划)的一部分。为了管理所有的团队，我们依赖于 GitHub 组织中的角色和团队。
+MDN Web 文档项目的成功和发展在很大程度上要归功于我们的贡献者社区。一些贡献者投入了部分时间，协助完成 MDN Web 文档的日常运行任务。对网站的修改，包括维护任务，都是由员工、承包商和合作伙伴网络完成的，他们都致力于 MDN Web 文档的健康、发展和维护。该项目在很大程度上依赖于 [GitHub 上的 MDN 组织](https://github.com/mdn)中的[角色](#角色)和[团队](#团队)来管理和整合来自这些不同群体的变更。该组织的成员名单可[在此找到](https://github.com/orgs/mdn/people)。
 
-在[这里可以查看](https://github.com/orgs/mdn/people)组织的成员清单。
+社区贡献对这个开源项目帮助巨大。贡献者可以通过他们在 MDN Web 文档上的工作展示他们的写作、技术和协作技能，以及与来自不同背景的人合作的能力。本节将介绍您在 MDN Web 文档项目志愿工作中可以担任的角色。
```
Comment (yin1999) [inline files/zh-cn/mdn/community/roles_teams/index.md:10]:
```suggestion
社区贡献对这个开源项目帮助巨大。贡献者可以通过他们在 MDN Web 文档上的工作展示他们的写作、技术和协作技能，以及与来自不同背景的人合作的能力。本节将介绍你在 MDN Web 文档项目志愿工作中可以担任的角色。
```

File: files/zh-cn/mdn/community/roles_teams/index.md [hunk 5]
```diff
@@ -5,111 +5,277 @@ slug: MDN/Community/Roles_teams
...
-  - **Yari content PT-BR**：负责审查 MDN 巴西葡萄牙语文档
-  - **Yari content RU**：负责审查 MDN 俄语文档
-  - **Yari content ZH**：负责审查 MDN 中文文档
+贡献者，或*社区参与者*，用他们的时间、技能、意见和想法为项目添砖加瓦。他们直接参与项目工作，并为项目增添价值。除了撰写并测试代码外，创建和更新文档、调查研究、修复缺陷和帮助其他社区成员都是贡献的一种。
 
-### 加入团队意味着什么
+根据贡献频率不同，你可以是偶尔贡献的人，也可以是积极贡献的人。如果你对项目产生了重大影响，可能会被提名为[杰出贡献者](#杰出贡献者)或晋升为[组织成员](#组织成员)。
 
-我们使用 [GitHub 团队](https://docs.github.com/zh/organizations/organizing-members-into-teams/about-teams)功能来管理团队。这意味着，当你被加入到团队中时，你已经传达了更密切地参与项目的意图。这也意味着你有一些额外的责任和权利：
+如果你是新人，并且想成为一名贡献者，可以看看我们的[贡献者指南](https://github.com/mdn/content/blob/main/CONTRIBUTING.md)和 [MDN GitHub 组织中的仓库列表](https://github.com/orgs/mdn/repositories)。
```
Comment (yin1999) [inline files/zh-cn/mdn/community/roles_teams/index.md:24]:
```suggestion
如果你是新人，并且想成为一名贡献者，可以看看我们的[贡献指南](https://github.com/mdn/content/blob/main/CONTRIBUTING.md)和 [MDN GitHub 组织中的仓库列表](https://github.com/orgs/mdn/repositories)。
```

File: files/zh-cn/mdn/community/roles_teams/index.md [hunk 6]
```diff
@@ -5,111 +5,277 @@ slug: MDN/Community/Roles_teams
...
-3. 团队成员具有更高的仓库访问权限。仅为成员分配需要的仓库的访问权限。
+作为一名贡献者，你可以通过参与以下活动来参与该项目：
 
-### 如何加入团队
+- 在[沟通渠道](/zh-CN/docs/MDN/Community/Communication_channels)中参与社区讨论。
+- 帮助其他贡献者处理拉取请求和问题，或指导新贡献者。
+- 提交缺陷报告。查看 MDN [`content`](https://github.com/mdn/content) 仓库中[你可以提出议题的类型](https://github.com/mdn/content/issues/new/choose)。如果你发现了平台的缺陷，你可以在 MDN 的 `yari` 仓库中[提出议题](https://github.com/mdn/yari/issues/new/choose)。
+- 对问题发表评论，推动对话取得丰硕成果。
+- 通过提交[拉取请求](/zh-CN/docs/MDN/Community/Pull_requests)来解决开启的议题（例如，在 [`content`](https://github.com/mdn/content/issues) 仓库中）。
+- 出席社区活动。
```
Comment (yin1999) [inline files/zh-cn/mdn/community/roles_teams/index.md:33]: How about "参与"

```suggestion
- 参与社区活动。
```

File: files/zh-cn/mdn/community/roles_teams/index.md [hunk 7]
```diff
@@ -5,111 +5,277 @@ slug: MDN/Community/Roles_teams
...
+- 帮助推荐 MDN 项目。
 
-要成为团队成员，你需要：
+**需求：**
```
Comment (yin1999) [inline files/zh-cn/mdn/community/roles_teams/index.md:36]: I guess the tone of "需求" is relatively weak, should we use "要求"?

```suggestion
**要求：**
```

File: files/zh-cn/mdn/community/roles_teams/index.md [hunk 8]
```diff
@@ -5,111 +5,277 @@ slug: MDN/Community/Roles_teams
...
 
-- 同意遵守我们的[社区参与准则](https://www.mozilla.org/zh-CN/about/governance/policies/participation/)。
-- 同意 Mozilla 的[提交访问权限的要求](https://www.mozilla.org/zh-CN/about/governance/policies/commit/requirements/)（Commit Access Requirement）。
-- 在你的 GitHub 账户上配置[双重身份验证](https://docs.github.com/zh/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication)。
+如果想要成为一名贡献者，你需要遵循：
 
-如果你想成为团队的一员，你应首先在[这个仓库提交一个议题](https://github.com/mdn/mdn/issues/new/choose)。
+- [Mozilla 行为准则](https://github.com/mdn/mdn-community/blob/main/CODE_OF_CONDUCT.md)
+- [MDN 的贡献准则](https://github.com/mdn/mdn-community/blob/main/CONTRIBUTING.md)
```
Comment (yin1999) [inline files/zh-cn/mdn/community/roles_teams/index.md:41]: This link is broken, and it seems that the file was never created in the mdn/mdn-community repository. Hey @bsmth. Is there any plan to creating this, or should we state that this file is located in the root directory of each repo.

https://github.com/mdn/content/blob/27e8dec205645aa69d3171e7004191a6066debf3/files/en-us/mdn/community/roles_teams/index.md?plain=1#L48-L51

File: files/zh-cn/mdn/community/roles_teams/index.md [hunk 9]
```diff
@@ -5,111 +5,277 @@ slug: MDN/Community/Roles_teams
...
+
+**权利：**
 
-## 所有者
+维护者有权批准并合并拉取请求。
 
-所有者拥有广泛的权限来管理用户和团队、维护 MDN 组织中跨仓库的访问权限、维护仓库设置，以及部署产品到生产环境。所有者目前仅限于 Mozilla 员工，而且受到其他的用户级别的所有要求的约束。除了其他用户级别的所有责任外，所有者还有以下额外的责任：
+### 所有者
+
+所有者拥有管理用户和 [GitHub 团队](https://github.com/orgs/mdn/teams)、维护 [MDN 组织](https://github.com/mdn)中各版本库的访问权限、维护版本库设置以及部署到生产环境的广泛权限。所有者受其他贡献者角色的所有要求约束。
```
Comment (yin1999) [inline files/zh-cn/mdn/community/roles_teams/index.md:104]:
```suggestion
所有者拥有管理用户和 [GitHub 团队](https://github.com/orgs/mdn/teams)、维护 [MDN 组织](https://github.com/mdn)中各仓库的访问权限、维护仓库设置以及部署到生产环境的广泛权限。所有者受其他贡献者角色的所有要求约束。
```

File: files/zh-cn/mdn/community/roles_teams/index.md [hunk 10]
```diff
@@ -5,111 +5,277 @@ slug: MDN/Community/Roles_teams
...
+
+> **备注：** 所有者角色目前仅限 Mozilla 员工获得。
```
Comment (yin1999) [inline files/zh-cn/mdn/community/roles_teams/index.md:106]:
```suggestion
> **备注：** 所有者角色目前仅限于 Mozilla 员工。
```

File: files/zh-cn/mdn/community/roles_teams/index.md [hunk 11]
```diff
@@ -5,111 +5,277 @@ slug: MDN/Community/Roles_teams
...
 
-此外，所有者应该：
+**权利：**
+
+所有者可以：
 
 - 根据需要添加和删除组织所有者和成员。
-- 根据需要添加仓库（作为新的项目或转移仓库）。
+- 根据需要向特定存储库添加和删除协作者。
+- 根据需要添加版本库（作为新项目或转移）。
```
Comment (yin1999) [inline files/zh-cn/mdn/community/roles_teams/index.md:124]:
```suggestion
- 根据需要添加仓库（作为新项目或转移）。
```

File: files/zh-cn/mdn/community/roles_teams/index.md [hunk 12]
```diff
@@ -5,111 +5,277 @@ slug: MDN/Community/Roles_teams
...
+- 确定并实施贡献者入职体验。
+- 对新的贡献者和用户进行入职培训。
+- 确保 MDN 项目和所有参与者的健康和福利。
+- 确定并协助实施自动化，以提高项目的可持续性。
+- 会见并确保与贡献者和合作伙伴保持健康的关系。
+- 在适当的情况下，协助问题分流和拉动请求审查。
+- 监控所有[沟通渠道](/zh-CN/docs/MDN/Community/Communication_channels)。
+- 表彰工作出色和/或为 MDN 项目做出贡献的贡献者。
+
+## 过程
```
Comment (yin1999) [inline files/zh-cn/mdn/community/roles_teams/index.md:191]:
```suggestion
## 流程
```

File: files/zh-cn/mdn/community/roles_teams/index.md [hunk 13]
```diff
@@ -5,111 +5,277 @@ slug: MDN/Community/Roles_teams
...
+
+1. 在 `mdn/mdn` 仓库的 `Issues` 选项卡中，点击右侧的 [**New issue**](https://github.com/mdn/mdn/issues/new/choose) 按钮。
+2. 在“Nominate a maintainer”（提名维护者）下，点击 **Get Started** 按钮。
+3. 在表单中填写你要提名的人员的贡献细节并提交表单。
+
+### 提名杰出贡献者
+
+查看作为[杰出贡献者](#杰出贡献者)的条件。
+
+想要提名某人为杰出贡献者，在 GitHub 上开启议题：
```
Comment (yin1999) [inline files/zh-cn/mdn/community/roles_teams/index.md:207]: Cloud you also split those lines for the en-US version?

https://github.com/mdn/content/blob/27e8dec205645aa69d3171e7004191a6066debf3/files/en-us/mdn/community/roles_teams/index.md?plain=1#L242-L243

File: files/zh-cn/mdn/community/roles_teams/index.md [hunk 14]
```diff
@@ -5,111 +5,277 @@ slug: MDN/Community/Roles_teams
...
+- `@yari-content`：MDN Web 文档所有的内容审查团队的总称
+
+  - 在不同的领域下，如无障碍、插件、CSS、HTML、HTTP、JavaScript、SVG、Web API 和 WebAssembly 下会存在很多子团队。如 `@yari-content-css` 和 `@yari-content-svg`。
+  - 在不同本地化语言下，如巴西葡萄牙语、汉语、法语、日语、韩语、俄语和西班牙语下也会存在很多子团队。如 `@yari-content-fr` 和 `@yari-content-ko`。
+
+为了成为团队中的一员，你必须：
+
+- 同意遵守我们的[社区参与准则](https://www.mozilla.org/zh-CN/about/governance/policies/participation/)。
+- 同意 Mozilla 的[提交访问要求](https://www.mozilla.org/zh-CN/about/governance/policies/commit/requirements/)。
+- 在 GitHub 账户上设置[双因素身份验证](https://docs.github.com/zh/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication)（2FA）。
```
Comment (yin1999) [inline files/zh-cn/mdn/community/roles_teams/index.md:277]: Could we align the translation of `2FA`. There are two translations above that differ from this. I prefer "双因素身份验证"

File: files/zh-cn/mdn/community/roles_teams/index.md [hunk 15]
```diff
@@ -5,111 +5,277 @@ slug: MDN/Community/Roles_teams
...
+- 至少为一个项目领域做出了积极贡献。
 
-作为特邀专家小组的成员，会被：
+以下两个需求是必须满足的：
 
-- 受邀参与每周的 MDN Web 文档编辑电话会议。
-- 受邀加入 Matrix 上的特邀专家频道。
-- 加入到相关主题团队，以自动分配拉取请求的审查。
-- 授予他们与审查拉取请求的相关仓库的合并权限。
+- 在你的 GitHub 账户开启[二步认证](https://docs.github.com/zh/authentication/securing-your-account-with-two-factor-authentication-2fa)。
```
Comment (jasonren0403) [inline files/zh-cn/mdn/community/roles_teams/index.md:64]:
```suggestion
- 在你的 GitHub 账户开启[双因素身份验证](https://docs.github.com/zh/authentication/securing-your-account-with-two-factor-authentication-2fa)。
```

File: files/zh-cn/mdn/community/roles_teams/index.md [hunk 16]
```diff
@@ -5,111 +5,277 @@ slug: MDN/Community/Roles_teams
...
-- 根据需要添加仓库（作为新的项目或转移仓库）。
+- 根据需要向特定存储库添加和删除协作者。
+- 根据需要添加仓库（作为新项目或转移）。
+
+### 角色总结
+
+| 角色                      | 需求                                                                                                                                                               | 权利                                                   |
+| :------------------------ | :----------------------------------------------------------------------------------------------------------------------------------------------------------------- | :----------------------------------------------------- |
+| [**贡献者**](#贡献者)     | 遵守行为准则和贡献准则                                                                                                                                             | - 受邀请参加贡献者活动 <br> - 有资格成为组织成员       |
+| [**组织成员**](#组织成员) | - 在 GitHub 账户上启用二步认证（2FA）<br> - 启用签名提交<br><br>至少一个：<br>- 解决至少两个议题<br> - 至少贡献了两个月<br>- 在一个项目领域作出活跃贡献            | 组织级别的访问权限                                     |
```
Comment (jasonren0403) [inline files/zh-cn/mdn/community/roles_teams/index.md:131]:
```suggestion
| [**组织成员**](#组织成员) | - 在 GitHub 账户上启用双因素身份验证（2FA）<br> - 启用签名提交<br><br>至少一个：<br>- 解决至少两个议题<br> - 至少贡献了两个月<br>- 在一个项目领域作出活跃贡献            | 组织级别的访问权限                                     |
```

File: files/zh-cn/mdn/community/roles_teams/index.md [hunk 17]
```diff
@@ -5,111 +5,277 @@ slug: MDN/Community/Roles_teams
...
+- 有资格成为[组织成员](#组织成员)
 
-## 特邀专家计划
+### 组织成员
 
-在 MDN Web 文档中，我们承诺为开放的 web 内容撰写文档。这不仅意味者要编写和维护大量不断变化发展的文档，而且我们难以对开放的 web 中的所有领域都有深入的了解。为了填补空白，我们启动了*特邀专家*计划。
+[组织成员](https://github.com/orgs/mdn/people)是受到认可的[贡献者](#贡献者)，他们定期参与 MDN Web 文档项目并作出贡献。他们的行为应符合项目的利益。
 
-特邀专家是在特定领域具有深入专业知识，并由社区、我们的合作伙伴或 MDN Web 文档内部成员提名的个人。
+**需求：**
```
Comment (jasonren0403) [inline files/zh-cn/mdn/community/roles_teams/index.md:54]:
```suggestion
**要求：**
```

File: files/zh-cn/mdn/community/roles_teams/index.md [hunk 18]
```diff
@@ -5,111 +5,277 @@ slug: MDN/Community/Roles_teams
...
+
+- 确定所负责项目的优先事项。
+- 参加社区会议。
+- 指导所有其他角色的新老贡献者。
+- 根据所掌握的技能，在项目领域提出、批准或实施建议：
+  - 代码和基础设施改进
+  - 内容改进
+  - 流程改进
+
+**需求：**
```
Comment (jasonren0403) [inline files/zh-cn/mdn/community/roles_teams/index.md:85]:
```suggestion
**要求：**
```

File: files/zh-cn/mdn/community/roles_teams/index.md [hunk 19]
```diff
@@ -5,111 +5,277 @@ slug: MDN/Community/Roles_teams
...
+维护者有权批准并合并拉取请求。
 
-所有者拥有广泛的权限来管理用户和团队、维护 MDN 组织中跨仓库的访问权限、维护仓库设置，以及部署产品到生产环境。所有者目前仅限于 Mozilla 员工，而且受到其他的用户级别的所有要求的约束。除了其他用户级别的所有责任外，所有者还有以下额外的责任：
+### 所有者
+
+所有者拥有管理用户和 [GitHub 团队](https://github.com/orgs/mdn/teams)、维护 [MDN 组织](https://github.com/mdn)中各仓库的访问权限、维护仓库设置以及部署到生产环境的广泛权限。所有者受其他贡献者角色的所有要求约束。
+
+> **备注：** 所有者角色目前仅限于 Mozilla 员工。
+
+**需求：**
```
Comment (jasonren0403) [inline files/zh-cn/mdn/community/roles_teams/index.md:108]:
```suggestion
**要求：**
```

File: files/zh-cn/mdn/community/roles_teams/index.md [hunk 20]
```diff
@@ -5,111 +5,277 @@ slug: MDN/Community/Roles_teams
...
+
+除了作为[组织成员](#组织成员)的职责外，特邀专家还需要：
+
+- 遵循[审查指南](https://github.com/mdn/content/blob/main/REVIEWING.md)。
+- 审查有关他们研究主题下的拉取请求。
+- 帮助其他贡献者成为审查员。
+
+当拉取请求在受邀专家的主题领域被提出时，会自动分配专家进行审查。如果一个主题领域有多个专家，则会使用[负载平衡策略](https://docs.github.com/zh/organizations/organizing-members-into-teams/managing-code-review-settings-for-your-team#about-auto-assignment)
+
+**需求：**
```
Comment (jasonren0403) [inline files/zh-cn/mdn/community/roles_teams/index.md:157]:
```suggestion
**要求：**
```

File: files/zh-cn/mdn/community/roles_teams/index.md (hunk 13 above)
Comment (jasonren0403) [reply files/zh-cn/mdn/community/roles_teams/index.md:207]: PR raised

File: files/zh-cn/mdn/community/roles_teams/index.md [hunk 21]
```diff
@@ -5,111 +5,277 @@ slug: MDN/Community/Roles_teams
...
 
-- 同意遵守我们的[社区参与准则](https://www.mozilla.org/zh-CN/about/governance/policies/participation/)。
-- 同意 Mozilla 的[提交访问权限的要求](https://www.mozilla.org/zh-CN/about/governance/policies/commit/requirements/)（Commit Access Requirement）。
-- 在你的 GitHub 账户上配置[双重身份验证](https://docs.github.com/zh/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication)。
+- 开启了至少两个拉取请求并获得合并，它们解决了至少两个议题。
+- 已经为 MDN Web 文档项目贡献了至少两个月。
+- 至少为一个项目领域做出了积极贡献。
 
-作为特邀专家小组的成员，会被：
+以下两个需求是必须满足的：
```
Comment (yin1999) [inline files/zh-cn/mdn/community/roles_teams/index.md:62]:
```suggestion
以下两个要求是必须满足的：
```

**Review by yin1999 — APPROVED** [review]
LGTM. Thanks @jasonren0403. Let's wait for the response from bsmth before merging :)

File: files/zh-cn/mdn/community/roles_teams/index.md (hunk 8 above)
Comment (bsmth) [reply files/zh-cn/mdn/community/roles_teams/index.md:41]: >Is there any plan to creating this, or should we state that this file is located in the root directory of each repo.

There's no plan to create this, no: we're already linking to code of conduct (Community Participation Guidelines) so contribution guide should be per project `CONTRIBUTING.md`. I'm fine with saying 'check the `CONTRIBUTING.md` file in each repo for guidance' as you suggested 👍🏻

Comment (yin1999) [reply files/zh-cn/mdn/community/roles_teams/index.md:41]: @jasonren0403 could you also fix this ;)

Comment (jasonren0403) [reply files/zh-cn/mdn/community/roles_teams/index.md:41]: > @jasonren0403 could you also fix this ;)

Should we point to `translated-content` in localized content, and `content` in en-US content? 🤔 

```suggestion
- [MDN 的贡献准则](https://github.com/mdn/translated-content/blob/main/CONTRIBUTING.md)
```

Comment (yin1999) [reply files/zh-cn/mdn/community/roles_teams/index.md:41]: Maybe something like this:

```md
- Contribution guidelines (check the `CONTRIBUTING.md` file in each repo)
```

**Review by yin1999 — APPROVED** [review]
Thanks @jasonren0403 and @bsmth :)

**Comment by bsmth** [comment]
Thanks, both!

**Pull request merged**